PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/HostJsonPatcher.kt

```diff
@@ -42,26 +42,34 @@ object HostJsonPatcher {
         return candidates.firstOrNull { it.exists() }?.canonicalFile
     }
 
-    fun tryPatchHostJsonFile(workingDirectory: String, functionNames: String) {
+    fun tryPatchHostJsonFile(workingDirectory: String, functionNames: String) =
+            tryPatchHostJsonFile(
+                    hostJsonFile = determineHostJsonFile(workingDirectory),
+                    functionNames = functionNames.split(',', ';', '|', ' '))
 
-        val hostJsonFile = determineHostJsonFile(workingDirectory)
+    private fun tryPatchHostJsonFile(hostJsonFile: File?, functionNames: List<String>) {
 
-        if (hostJsonFile == null) {
+        if (hostJsonFile == null || !hostJsonFile.exists()) {
```

Comment:
No changes needed, just checking - this should be already covered by `determineHostJsonFile` call.